### PR TITLE
Use espresso arbitrum sdk

### PR DESCRIPTION
**Pull request by sveitser**
Use our fork of `arbitrum-sdk`

### CODE REVIEW

File: packages/arb-token-bridge-ui/next.config.js [hunk 1]
```diff
@@ -5,6 +5,11 @@
  **/
 
 module.exports = {
+  basePath: '/arbitrum-token-bridge',
```
Comment (sveitser) [inline packages/arb-token-bridge-ui/next.config.js]: I think this should be removed when we have a custom domain

File: packages/arb-token-bridge-ui/src/pages/index.tsx [hunk 2]
```diff
@@ -77,49 +77,49 @@ function addOrbitChainsToArbitrumSDK() {
...
-        destination: getDestinationWithSanitizedQueryParams(sanitized, query)
-      }
-    }
-  }
-
-  return {
-    props: {}
-  }
-}
+// export function getServerSideProps({
```
Comment (sveitser) [inline packages/arb-token-bridge-ui/src/pages/index.tsx]: This is commented out to make next generate a static page. Maybe there's a better way.

File: .github/workflows/espresso-pages.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID }}
+          NEXT_PUBLIC_SEPOLIA_RPC_URL: https://ethereum-sepolia-rpc.publicnode.com
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v4
+        if: ${{ github.ref == 'refs/heads/integration' }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./packages/arb-token-bridge-ui/build
+          cname: arbitrum-bridge.cappuccino.testnet.espresso.network
```
Comment (sveitser) [inline .github/workflows/espresso-pages.yml]: @Ancient123 do you think this is a reasonable domain for the arbitrum bridge frontend for our deployment?

**Comment by sveitser** [comment]
I gave up on using github pages. The deployment is broken in subtle ways.

**Comment by sveitser** [comment]
@Ancient123 can we deploy this with vercel?

It needs this override

Screenshot 2024-08-12 at 22 27 08

And the env vars `NEXT_PUBLIC_SEPOLIA_RPC_URL=https://ethereum-sepolia-rpc.publicnode.com` and `NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID`, which can be obtained at https://cloud.walletconnect.com/sign-up, or I can send you one I created.

**Review by Sneh1999 — APPROVED** [review]

**Review by ImJeremyHe — APPROVED** [review]

**Pull request merged**